config/manifests/bases/awx-operator.clusterserviceversion.yaml

```
@@ -55,6 +55,11 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:hidden
+      - displayName: PostgreSQL Security Context Settings
+        path: postgres.security_context_settings
```

The name you chose for this parameter is good because it fits with the current V1 AWX schema (`postgres_security_context_settings`), however the reference here will not resolve correctly.  

It should be:
```suggestion
        path: postgres_security_context_settings
```